### DISCUSSION TITLE
Matplotlib plt.show() only works once

### DISCUSSION THREAD

**Discussion by Duumbo** [xonsh/xonsh#5681] [Q&A]
Hi,
When using matplotlib in the xonsh shell, I can only show the figures once per shell session. Any idea on how to fix this bug?
I am using arch so my python installation is managed by pacman, but I usually use virtualenv to install my packages for different projects.

**Comment by anki-code** [comment]
May be diving into https://github.com/xonsh/xontrib-mpl code can help.
